src/api.ts

```diff
@@ -269,7 +269,7 @@ export async function generateApi (file: SourceFile, spec: OpenAPIV3.Document, o
       writer.writeLine('const ret: Pick<T, K> = {} as Pick<T, K>;')
       writer.write('keys.forEach(key => {')
       writer.indent(() => {
-        writer.writeLine('if (key in obj)')
+        writer.writeLine('if (obj && obj[key])')
```

Comment:
```suggestion
        writer.writeLine('if (Object.keys(obj).includes(key))')
```
to handle null/undefined values

Reply:
done